data/management/commands/create_org_cluster_map_and_activity_graph.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import json
+import getorg
+import os
+
+from django.core.management.base import BaseCommand
+
+from data.models import Contributor
+from activity.scraper import activity_json
+
+
+class Command(BaseCommand):
+    help = 'Create a cluster map using contributors geolocation'
+
+    def add_arguments(self, parser):
+        parser.add_argument('output_dir', nargs='?', type=str)
+
+    def handle(self, *args, **options):
```

Comment:
`handle` should have only few lines of code, add all the other code in a separate file and use them here. E.g. see https://github.com/coala/community/blob/master/meta_review/management/commands/run_meta_review_system.py